client/source/class/cv/io/Client.js

```diff
@@ -475,16 +498,30 @@ qx.Class.define('cv.io.Client', {
             delete options.beforeSend;
           }
           if (options.listeners) {
-            Object.getOwnPropertyNames(options.listeners).forEach(function(eventName) {
-              var qxEventName = eventName !== 'error' ? eventName : 'statusError';
-              ajaxRequest.addListener(qxEventName, options.listeners[eventName], context);
+            Object.getOwnPropertyNames(options.listeners).forEach(function (
+              eventName
+            ) {
```

Comment:
Can we avoid this type of change?

I think
```
            Object.getOwnPropertyNames(options.listeners).forEach(function (eventName) {
``` 
is far more readable than
```
            Object.getOwnPropertyNames(options.listeners).forEach(function (
              eventName
            ) {
```

Reply:
All I can do is changing the allowed line width (default 80) to e.g. 120 to avoid some line breaks. Other options to control this kind of formatting do not seem to be possible, at least I have found nothing in the prettier documentation.

Prettier calls iself an "opinionated code formatter" which implies that you don't have a lot of options to configure it for your own needs.